libraries/AP_DDS/AP_DDS_Service_Table.h

```diff
@@ -51,6 +54,18 @@ constexpr struct AP_DDS_Client::Service_table AP_DDS_Client::services[] = {
         .reply_topic_name = "rr/ap/mode_switchReply",
     },
 #endif // AP_DDS_MODE_SWITCH_SERVER_ENABLED
+#if AP_DDS_COPTER_TAKEOFF_SERVER_ENABLED
+{
+        .req_id = to_underlying(ServiceIndex::TAKEOFF),
+        .rep_id = to_underlying(ServiceIndex::TAKEOFF),
+        .service_rr = Service_rr::Replier,
+        .service_name = "rs/ap/takeoffService",
```

Comment:
Because some disagreement from the dev team on whether this should be relative to home or relative to takeoff altitude, and whether we should be using guided topic, can we add the `experimental` prefix to this so we can change the interface?

Reply:
Hi @Ryanf55 ! You mean "rs/ap/experimental/takeoffService", or AP_DDS_EXPERIMENTAL_ENABLED?

Reply:
What about relative to terrain?

Reply:
> What about relative to terrain?

The IDL file has it relative to terrain IF rangefinder is in use.